packages/account-v2/src/containers/TwoFactorAuthenticationForm/TwoFactorAuthenticationForm.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useRef } from 'react';
+import { Form, Formik, FormikProps } from 'formik';
+import { useTwoFactorAuthentication, useTwoFactorAuthenticationStatus } from '@deriv/api-v2';
+import { InputWithButton } from '../../components/InputWithButton';
+import { API_ERROR_CODES, getTwoFactorAuthenticationFormValidationSchema } from '../../utils';
+
+type TTwoFactorData = {
```

Comment:
Type can be inferred from the Yup Schema `Yup.InferType<ReturnType<typeof getTwoFactorAuthenticationFormValidationSchema>>` or `Yup.InferType<typeof validationSchema>`